projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.ts

```diff
@@ -10,28 +10,97 @@ import { IFeedItem } from 'projects/shared-models/feed-item.model';
 })
 export class ExternalFeedComponent implements OnInit {
   externalPosts: IFeedItem[] = [];
+  externalPostsV2 = [];
```

Comment:
We don't need this, the earlier variable (externalPosts) can be used

Reply:
Removed